components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -5384,4 +5387,180 @@ protected String getSearchQuery(String searchQuery) throws APIManagementExceptio
         }
         return criteria;
     }
+
+    /**
+     * Method to get the user specified mediation sequence.
+     *
+     * @param apiIdentifier : The identifier of the api.
+     * @param type : Mediation type. {in, out, fault}
+     * @param name : The name of the sequence that needed.
+     * @return : The content of the mediation sequence.
+     */
+    public String getSequenceFile(APIIdentifier apiIdentifier, String type, String name) throws APIManagementException {
+
+        Resource requiredSequence;
+        InputStream sequenceStream;
+        String sequenceText = "";
+
+        try {
+            if (apiIdentifier != null && type != null && name != null) {
+                requiredSequence = getDefaultSequence(type, name);
+                if (log.isDebugEnabled()) {
+                    log.debug("Get the default sequences.");
+                }
+                if (requiredSequence == null) {
+                    if (log.isDebugEnabled()) {
+                        log.debug("Get the custom sequences.");
+                    }
+                    requiredSequence = getCustomSequence(apiIdentifier, type, name);
+                }
+
+                //Convert the content stream to a string.
+                if (requiredSequence != null) {
+                    sequenceStream = requiredSequence.getContentStream();
+                    StringWriter stringWriter = new StringWriter();
+                    IOUtils.copy(sequenceStream, stringWriter);
+                    sequenceText = stringWriter.toString();
+                } else {
+                    log.error("No sequence for the name " + name + "is found!");
```

Comment:
you will need space before is found

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/5114/commits/726f591adc11d8584c2c1e375a6a5add578d90ed